.github/workflows/merge.yml

```diff
@@ -54,11 +54,17 @@ jobs:
           pip install --upgrade --no-input pip
           pip install --upgrade --no-input protobuf
 
-      - name: Run tests
+      - name: Run unit tests
```

Comment:
why not spin up any required services as part of the `unit-test` make target, so that they can be run as easily on the console as in a GHA.

Reply:
👍  updated.